gcp-gmp-integration.md

```diff
@@ -0,0 +1,69 @@
+
+For this integration, GMP is required to be enabled for your GKE cluster with the managed collection. Next, Kubecost is installed in your GKE cluster and leverages GMP Prometheus binary to ingest metrics into GMP database seamlessly. In this setup, Kubecost deployment also automatically creates a Prometheus proxy that allows Kubecost to query the metrics from the GMP database for cost allocation calculation.
+
+{% hint style="info" %}
+This integration is currently in beta.
+{% endhint %}
+
+## Reference Resources
+
+* [Google Cloud Managed Service for Prometheus (GMP)](https://cloud.google.com/stackdriver/docs/managed-prometheus)
+* [Get started with self-deployed collection](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-unmanaged)
+* [Configure a query interface](https://cloud.google.com/stackdriver/docs/managed-prometheus/query)
+
+## Instructions
+
+### Prerequisites
+
+* You have a GCP account/subscription.
+* You have permission to manage GKE clusters and GCP monitoring services.
+* You have an existing GKE cluster with GMP enabled. You can learn more [here](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#enable-mgdcoll-gke).
+
+### Installations
+
+You can use the following command to install Kubecost on your GKE cluster and integrate with GMP:
+
+```bash
+export GCP_PROJECT_ID="YOUR_GCP_PROJECT_ID"
+export CLUSTER_NAME="YOUR_GKE_CLUSTER_NAME"
+helm upgrade -i kubecost cost-analyzer/ \
+--namespace kubecost --create-namespace \
+--set prometheus.server.image.repository="gke.gcr.io/prometheus-engine/prometheus" \
+--set prometheus.server.image.tag="v2.35.0-gmp.2-gke.0" \
+--set global.gmp.enabled="true" \
+--set global.gmp.gmpProxy.projectId="${GCP_PROJECT_ID}" \
+--set prometheus.server.global.external_labels.cluster_id="${CLUSTER_NAME}" \
+--set kubecostProductConfigs.clusterName="${CLUSTER_NAME}"
+```
+In this installation command, these additional flags are added to have Kubecost work with GMP:
+
+- `prometheus.server.image.repository` and `prometheus.server.image.tag` replace the standard Prometheus image by GMP specific image.
+- `global.gmp.enabled` and `global.gmp.gmpProxy.projectId` are for enabling the GMP integration.
+- `prometheus.server.global.external_labels.cluster_id` and `kubecostProductConfigs.clusterName` helps to set name for your Kubecost setup
```

Comment:
Similar grammar tweak suggestion: s/to set name for/to set the name for

Also, the engineer in me needs every bulleted list item to either end with punctuation or not. Please either put a period at the end of this line or remove the period from the previous 2 bullet points :)

Reply:
Thanks, fixed in the latest commit.